src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LoggingKeys.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Loggers
+{
+    internal static class LoggingKeys
+    {
+        // These are publicly visible as property names or prefixes
```

Comment:
You have some extra alongside the code. Like `[Unknwon]` for the function name. Consider including here